server/test/todo.test.js

```diff
@@ -19,7 +19,8 @@ describe('Todo', function() {
         assert.equal(this.res.statusCode, 200);
       });
 
-      it('should respond with a new todo', function() {
+      it.skip('should respond with a new todo', function() {
+        console.log(this.res.body.id);
```

Comment:
`this.res.body.id` returns an actual MongoDB id instead of the `t-math` id. Any idea why this is happening?